CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
-# @maintainers will be requested for review when someone
-# opens a pull request.
-*       @projectcontour/maintainers
\ No newline at end of file
+# @maintainers and @reviewers will be requested for review when
+# someone opens a pull request.
+*       @projectcontour/maintainers @projectcontour/reviewers
```

Comment:
hm, the validation says this isn't valid since the reviewer team doesn't have write access as i feared